test/unit/durableclient-spec.ts

```diff
@@ -244,6 +244,49 @@ describe("Durable client RPC endpoint", () => {
             expect(scope.isDone()).to.be.equal(true);
             expect(result).to.be.an("array");
         });
+
+        it("uses continuation header", async () => {
+            const input = JSON.parse(durableClientBindingInputJson) as OrchestrationClientInputData;
+            const client = new DurableOrchestrationClient(input);
+
+            // The getStatusBy() method should do a GET to http://127.0.0.1:17071/durabletask/instances/?createdTimeFrom=2020-01-01T00:00:00Z&createdTimeTo=2020-01-01T23:59:59Z&runtimeStatus=Pending,Running,Completed,Terminated,Failed
+            const expectedUrl = new URL(`${testRpcOrigin}/durabletask/instances/`);
+            const createdTimeFrom = "2020-01-01T00:00:00.000Z";
+            const createdTimeTo = "2020-01-01T23:59:59.000Z";
+            const runtimeStatus = "Pending,Running,Completed,Terminated,Failed";
+
+            const scopeWithTokenResponse = nock(expectedUrl.origin)
+                .get(expectedUrl.pathname)
+                .query({ createdTimeFrom, createdTimeTo, runtimeStatus })
+                .reply(200, [null, null], { "x-ms-continuation-token": "myToken" });

```

Comment:
small nit: Could we at least fill these with dummy objects. Returning an array filled with `null` as a shortcut feels a bit dirty. Ideally we would fill it with actual JSON representations of the data, but I realize that is a lot of work.

Reply:
Of course, I agree the `null` was a little dirty.
Just replaced it with proper JSONs